src/utils/errorHandler.js

```diff
@@ -18,6 +21,13 @@ export function handleError(event, options = {}) {
     console.log(event);
   }
 
+  if (event && event.message === NETWORK_CONNECTION_ERROR) {
```

Comment:
Use `errors` as the namespace for more clarity.

```suggestion
  if (event && event.message === errors.NETWORK_ERROR) {
```

Reply:
used errors as namespace